packages/components/src/utils/strings.ts

```diff
@@ -71,6 +72,36 @@ export const normalizeTextString = ( value: string ): string => {
 		.replace( ALL_UNICODE_DASH_CHARACTERS, '-' );
 };
 
+/**
+ * Converts any string to kebab case.
+ * Backwards compatible with Lodash's `_.kebabCase()`.
+ * Backwards compatible with `_wp_to_kebab_case()`.
+ *
+ * @see https://lodash.com/docs/4.17.15#kebabCase
+ * @see https://developer.wordpress.org/reference/functions/_wp_to_kebab_case/
+ *
+ * @param {string} str String to convert.
+ * @return {string} Kebab-cased string
+ */
+export function kebabCase( str: any ) {
```

Comment:
I was a little confused as to whether we should use `any` type, but it looks like this function is supposed to accept all types.

Reply:
I think this is the right call, although I wonder if we should use `unknown` instead ?

Reply:
Indeed. Updated by [5625a86](https://github.com/WordPress/gutenberg/pull/56758/commits/5625a86eaf7c99877c7332decd39d7ce5203078e). I did some minor refactoring to fix the type error. The unit tests are passing locally, so the logic shouldn't change at all.